README.md

```diff
@@ -15,7 +15,8 @@ The App Center SDK for Unity supports the following platforms:
 * Android (5.0/API 21 or later)
 * UWP (Build 16299 or later)
 
-Also note that the App Center SDK for Unity is only available in C#.
+> Starting from SDK version 3.0.0 we no longer support using .NET 3.5 scripting runtime version  on UWP
+> and .NET scripting backend.

```

Comment:
Please do not mention versions here, just describe current state.
Also, both limitations are about UWP + double space before "on"